client/cli/src/params/import_params.rs

```diff
@@ -78,6 +78,10 @@ pub struct ImportParams {
 		default_value = "Log"
 	)]
 	pub tracing_receiver: TracingReceiver,
+
+	/// Enable wasm tracing
+	#[structopt(long = "wasm-tracing")]
```

Comment:
```suggestion
	#[structopt(long)]
```

The name of the attribute is automagically generated by structopt